nixos/modules/services/video/photonvision.nix

```diff
@@ -0,0 +1,70 @@
+{ config, pkgs, lib, ... }:
+
+with lib;
+let
+  cfg = config.services.photonvision;
+in
+{
+  options = {
+    services.photonvision = {
+      enable = mkEnableOption (mdDoc "Enable PhotonVision");
+
+      package = mkOption {
+        type = types.package;
+        default = pkgs.photonvision;
+        defaultText = literalExpression "pkgs.zerotierone";
```

Comment:
default Text looks like it was from a copied nix module, shouldn't this be photovision?

Reply:
Yeah, I got the CI error about not being able to refer to `pkgs` and just looked at a random module for how to fix it. Thanks for the catch.